### PR TITLE
Fix NuGet warnings with package references

### PR DESCRIPTION
**Pull request by JoeRobich**
Similar changes were made in the dev16.11 branches but this release branch wasn't part of the codeflow

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -119,7 +119,7 @@
     <MicrosoftVisualStudioCallHierarchyPackageDefinitionsVersion>15.8.27812-alpha</MicrosoftVisualStudioCallHierarchyPackageDefinitionsVersion>
     <MicrosoftVisualStudioCodeAnalysisSdkUIVersion>15.8.27812-alpha</MicrosoftVisualStudioCodeAnalysisSdkUIVersion>
     <MicrosoftVisualStudioComponentModelHostVersion>$(VisualStudioEditorNewPackagesVersion)</MicrosoftVisualStudioComponentModelHostVersion>
-    <MicrosoftVisualStudioCompositionVersion>16.5.13</MicrosoftVisualStudioCompositionVersion>
+    <MicrosoftVisualStudioCompositionVersion>16.9.20</MicrosoftVisualStudioCompositionVersion>
```
Comment (JoeRobich) [inline eng/Versions.props:122]: This package contains the installer scripts that allow the package to install in framework-style projects.

File: src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj [hunk 2]
```diff
@@ -147,6 +147,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Shell.Design" Version="$(MicrosoftVisualStudioShellDesignVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Language.NavigateTo.Interfaces" Version="$(MicrosoftVisualStudioLanguageNavigateToInterfacesVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Threading" Version="$(MicrosoftVisualStudioThreadingVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.Validation" Version="$(MicrosoftVisualStudioValidationVersion)" />
```
Comment (jaredpar) [inline src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj:150]: What feed do we get this package from?

**Review by jaredpar — APPROVED** [review]

File: src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj (hunk 2 above)
Comment (JoeRobich) [reply src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.csproj:150]: https://dev.azure.com/azure-public/vside/_packaging?_a=package&feed=vssdk%40Local&package=Microsoft.VisualStudio.Validation&protocolType=NuGet&version=17.0.18-alpha

**Comment by JoeRobich** [comment]
The test failure seems to be a Helix issue where the Helix task has completed but the watcher doesn't realize that it is complete. Merging to get packages published.

**Pull request merged**